snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -1414,10 +1415,10 @@ static void LastOrDefaultEx3()
         public static void LongCountEx1()
         {
             // <Snippet47>
-            string[] fruits = { "apple", "banana", "mango",
-                                  "orange", "passionfruit", "grape" };
+            string[] fruits = [ "apple", "banana", "mango",
+                                  "orange", "passionfruit", "grape" ];
 
-            long count = fruits.LongCount();
+            long count = fruits.Length;

```

Comment:
And again, the purpose of this sample is to demonstrate the `LongCount` extension method. Replacing it with the `Length` property defeats the purpose.